### PR TITLE
[ Feature request ] : [ Table ] Added  feature to highlight a word in the column if matched with searched term

### LINKED ISSUE #2743: Add search term highlighting to table widget
### Is your proposal related to a problem?
Dynamically mark search terms when a user searches inside a table widget this can be provided as a setting option inside table widget inspector

### Describe the solution you'd like
When a user searches for a text highlight it if the setting option is set to true

Ex: When a user searches for keyword "sarah" the text can be highlighted where the keyword matches

image

### PR DESCRIPTION
**Pull request by manishkushare**
Resolves #2743

### CODE REVIEW

**Comment by shubh22** [comment]
Task linked: [CU-85zrk4d2z Feature request : column highlighting on search to table](https://app.clickup.com/t/85zrk4d2z)

File: frontend/src/Editor/Components/Table/GenerateEachCellValue.jsx [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+        updateCellValue.current = e.target.value;
+        setHighlighterCells(false);
+      }}
+      onMouseLeave={(e) => {
+        e.persist();
+        if (!showHighlightedCells && updateCellValue.current !== cellValue && _.isEmpty(rowChangeSet)) {
+          updateCellValue.current = null;
+          setHighlighterCells(true);
+        }
+      }}
```
Comment (kavinvenkatachalam) [inline frontend/src/Editor/Components/Table/GenerateEachCellValue.jsx:39]: @manishkushare Why do we need this?

**Review by kavinvenkatachalam — COMMENTED** [review]
@sherfin94 Can you check this PR?

**Review by kavinvenkatachalam — CHANGES_REQUESTED** [review]
@manishkushare Highlight is not working when I search with default data. Could you please check that?

**Review by kavinvenkatachalam — APPROVED** [review]

**Review by kavinvenkatachalam — APPROVED** [review]

**Pull request merged**